lib/l10n/arb/app_pt-br.arb

```diff
@@ -0,0 +1,82 @@
+{
+  "language":"Português - Brasil",
```

Comment:
Change this file's name to `lapp_pt.arb`, (Sorry again for my previous comment, this one will work)